packages/sanity/src/core/bundles/components/panes/BundleActions.tsx

```diff
@@ -78,7 +83,34 @@ export function BundleActions(props: BundleActionsProps): ReactNode {
     setIsInVersion(true)
   }, [documentId, globalBundleId, documentStore.pair, documentType, newVersion, toast, title])
 
-  /** TODO what should happen when you add a version if we don't have the ready button */
+  const handleDiscardVersion = useCallback(async () => {
```

Comment:
This is the same logic as what we currently have on the release detail that allows you to discard document versions

Reply:
There was some discussion (either on slack or on linear) about how we aren't using document operations.
I considered moving this to somewhere that could be imported directly from both, but maybe since we are likely going to change this in the coming future we don't need to do much beyond the TODO?

Reply:
cc @bjoerge